src/py42/clients/cases.py

```diff
@@ -0,0 +1,175 @@
+from enum import Enum
+
+from py42.util import parse_timestamp_to_milliseconds_precision
+
+
+class CaseStatus(Enum):
```

Comment:
This does not need to inherit the Enum class, just look at other enums in py42 for examples.

Reply:
Here is an example: https://github.com/code42/py42/blob/master/src/py42/services/detectionlists/departing_employee.py#L16

Reply:
Addressed.